### PR TITLE
6mic 8mic fix

### PR DESCRIPTION
**Pull request by shivasiddharth**

### CODE REVIEW

File: seeed-voicecard [hunk 1]
```diff
@@ -144,6 +175,15 @@ fi
...
+#Force 3.5mm ('headphone') jack
+#   The Raspberry Pi 4, released on 24th Jun 2019, has two HDMI ports,
+#   and can drive two displays with audios for two users simultaneously,
+#   in a "multiseat" configuration. The earlier single virtual ALSA
+#   option for re-directing audio playback between headphone jack and HDMI
+#   via a 'Routing' mixer setting was turned off eventually to allow
+#   simultaneous usage of all 3 playback devices.
+if aplay -l | grep -q "bcm2835 ALSA"; then
+    amixer cset numid=3 1 || true
+fi
```
Comment (HinTak) [inline seeed-voicecard:188]: This is very wierd - you seems to have branched off a long while ago and now including changes which have happened upstream meanwhile. Should you rebase first?

**Review by HinTak — CHANGES_REQUESTED** [review]
Besides including changes already upstream'ed, I think you need to explain why you did what you did in the conf files.

So in generally you probably should do a `git rebase ...` first then a `git rebase -i ...` with some editing and do some writing...

If you can explain why you did what you did I can probably do the rebase...

File: seeed-voicecard (hunk 1 above)
Comment (shivasiddharth) [reply seeed-voicecard:188]: Oops sorry. This was intended to be merged with my master branch. Was working with Git windows application that has raised this pulled request. Sorry for the inconvinience.

**Comment by HinTak** [comment]
@shivasiddharth the config change is interesting but it needs a bit more explanations.

**Pull request closed without merging**